app/core/utils.py

```diff
@@ -41,19 +42,20 @@ def create_iqe(
     return iq
 
 
-def safe_call_api(api_method: Callable, **kwargs):
+def safe_call_sdk(api_method: Callable, **kwargs):
```

Comment:
Just flagging that this is what's creating issues with `test_reset_retry` ([here](https://github.com/groundlight/python-sdk/blob/219a525b82187b420de094027470249a72e14b9f/test/unit/test_detector_reset.py#L9)) from the SDK tests.